File: 2ndPeriod/aedsII/clike/ifelse/ifelse.cpp
#include <iostream>

using namespace std;

int main()
{
    int x = -12;

    cout << "Enter a number: ";
    cin >> x;

    if(x>0)
    {
        cout << x << " is positive";
    }
    else
    {
        cout << x << " is negative";
    }
    
    return 0;
}

File: 2ndPeriod/aedsII/clike/ifelse/ifelse.cs
using System;

class Main{
    static void Main(string[] args){
        int x;
        Console.WriteLine("Enter a number: ");
        x = Console.ReadLine();

        if(x == 0){
            Console.WriteLine(x + " is void");
        } else if(x<0) {
            Console.WriteLine(x + " is negative");
        } else {
            Console.WriteLine(x + " is positive");
        }
    }
}